communications.md

```diff
@@ -1,68 +1,89 @@
+Documentation - Consolidation: Asynchronous communication can become chaotic so it should be every participant's, at least the initiator's obligation to organize and summarize afterwards and write it in well-formatted documentation as a part of our internal knowledge base. At a bare minimum, copy the transcript of relevant discussions into a place for future reference or refinement to be retained in the knowledge base.
 
-## Transparent by Default
-
-Don't be embarrassed about asking "silly" questions. Try to always write your messages in a publically accessible (or at least a team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
+Documentation - Transparency: Don't be embarrassed about asking "silly" questions. Try to always write your messages in a publically accessible (or at least a team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
 
 A few exceptions:
 
 - When you communicate credentials or secrets, use direct messages, or very well-controlled private channels.
 - When you talk about subjects that are irrelevant to work, use direct messages (but to be honest, wouldn't one of the social media/chat apps fit better?).
 
-## Budget Communication Time
+#### - Internal & External Communication Tool: Gmail -
 
-For asynchronous communications, try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+Minimize Notifications: A busy email inbox is a contributing factor to interruptions. Instead of striving to achieve "Inbox Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. You may do the following:
 
-For synchronous communications:
+- Pin/star items that need future attention. Archive them as soon as they're done, or no longer needed.
+- Optionally snooze items that need future attention. Archive them as soon as they're done, or no longer needed.
+- Archive the rest.
+
+Opt-out: Use filters to avoid receiving emails sent to a shared email address that you are not interested in.
+
+#### - Project Management & Product Issue Documentation: Asana -
+
+Opt-in: Join projects that you are interested in or obligated to.
+
+Minimize Notifications: Similar to Slack, instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". Defer any information that is not work-related
+
+Documentation - Transparency: Provide as much detail as possible for other team members to gain a better understanding of the task or issue.
```

Comment:
maybe add some phrase to emphasize that this information is for others to opt-in